index.html

```diff
@@ -6009,27 +6009,30 @@ <h2 id="AudioWorklet-Examples">
   }
 
   process (inputs, outputs, parameters) {
-    // Note that the input will be downmixed to mono; however, if no inputs are
+    // Note that the input will be down-mixed to mono; however, if no inputs are
     // connected then zero channels will be passed in.
-    let buffer = inputs[0][0];
-    let bufferLength = buffer.length;
-    let sum = 0, x = 0, rms = 0;
-
-    // Calculated the squared-sum.
-    for (let i = 0; i &lt; bufferLength; ++i) {
-      x = buffer[i];
-      sum += x * x;
-    }
+    if (inputs.length &gt; 0) {
```

Comment:
This isn't right, you do need to check `inputs[0].length`.

When inputs are disconnected, this doesn't change the _number_ of inputs (which is determined at the time the node was created). A disconnected input still exists in the `inputs` array, it just exposes zero channels. This is important because a node could have two input ports, one of which has no incoming connections, but the other one does.